sop-website/docs/Data-Requests/Data-Requests.mdx

```diff
@@ -3,3 +3,75 @@ title: Data Requests
+    [Google Drive](https://drive.google.com/drive/folders/1bDBSF2pkx50kTyTXmWJYt1Ta6pgwh4HH) is where the recorded lectures and task 1 materials are located.
+    [Backbone Config](https://www.github.com/OHNLP/BackboneConfigurator) -> Releases -> Latest is the download location. The configurator will handle downloading all the other toolkit components for you (so long as you have an 
+    internet connection).
+
+3. Imaging data
+    >The ideal upload would be DICOM-formatted images and the accompanying metadata, which may include useful clinical and technical information. RSNA LOINC codes specifying the type of study would be useful to 
+    include. Imaging studies should have a corresponding entry in the PROCEDURE_OCCURENCE OMOP table. Thus imaging studies can be linked to the rest of the data using the procedure_occurence_id field. Specific 
+    instructions as to the naming of individual image files and the method to include a link between imaging studies PROCEDURE_OCCURENCE will be forthcoming.
+
+4.	Continuous Waveforms
+    >The ideal upload will consist of files in the raw formats we have discussed with you - BedMaster, Philips DWC, and so on. Data are requested in as close to source format as possible (this could mean .stp 
+    for sites using Bedmaster, .csv exports from MS SQL server for sites using DWC without additional processing, .h5 for sites that can produce hdf5 versions of the waveform files, WFDB for sites that cannot 
+    provide more proximal versions of the data. Waveform files should be devoid of personal identifiers, and associated with a person_id linkable to the OMOP EHR data. For sites providing .h5 files, please be in 
+    touch with the DA waveform group (Will Ashe), so h5 group structures are well communicated. 
+
+5.	Social Determinants of Health
+    >We request that the site make every effort to include the gender_source_value, race_source_value and ethnicity_source_value fields in the PERSON table, given the wide variety of source concepts available across sites and the limited range permissible in the OMOP corresponding concept_id’s. As noted in the OMOP documentation, the PERSON table should refer to biological sex. Alternative assignments, if available, should be documented in the OBSERVATION table. We also request that sites provide insight, whenever available, on how race, ethnicity and sex variables are collected. This should be included in the descriptive file accompanying each upload in the root folder. Other pre-computed SDoH variables (e.g. ADI) available in structured data should be included in the OBSERVATION table. We request that sites keep (or identify the means to get access to) local documentation of full patient addresses, zip codes, payor information, marital status and employment status, whenever available. The intent is that, in the future, tools will be provided to sites to compute SDoH indices locally, which then could be uploaded.
```

Comment:
Nitpick, but I don't think the apostrophe is needed for plural `concept_id`because there is nothing belonging to it (in the same way there is no apostrophe below for "patient study IDs"). I think it should be `concept_id`s (which admittedly looks kind of awkward!).